markdown/admin/logfiles.html.md.erb

```diff
@@ -0,0 +1,272 @@
+| [`log_truncate_on_rotation`](../reference/guc/parameter_definitions.html#log_truncate_on_rotation) |  Identify whether to truncate or append to the log file |
+
+#### <a id="logging_config_query"></a>Configuring Query Logging
+
+HAWQ exposes a set of server configuration parameters dealing exclusively with GPORCA query executor and optimizer logging.  These include:
+
+| Server Configuration Parameter     | Description     |
+|-----------------------------------|------------------------|
+|  [`debug_print_parse`](../reference/guc/parameter_definitions.html#debug_print_parse) |  Log the query parse tree |
+|  [`debug_print_plan`](../reference/guc/parameter_definitions.html#debug_print_plan) |  Log the query plan |
+|  [`debug_print_prelim_plan`](../reference/guc/parameter_definitions.html#debug_print_prelim_plan) |  Log the preliminary query plan |
+|  [`debug_print_rewritten`](../reference/guc/parameter_definitions.html#debug_print_rewritten) |  Log the query rewriter output |
+|  [`debug_print_slice_table`](../reference/guc/parameter_definitions.html#debug_print_slice_table) |  Log the query slice plan |
+|  [`log_executor_stats`](../reference/guc/parameter_definitions.html#log_executor_stats) |  Log query executor performance statistics |
+|  [`log_parser_stats`](../reference/guc/parameter_definitions.html#log_parser_stats) |  Log query parser performance statistics |
+|  [`log_planner_status`](../reference/guc/parameter_definitions.html#log_planner_stats) |  Log the performance statistics of the legacy query optimizer (planner) |
+|  [`log_statement_stats`](../reference/guc/parameter_definitions.html#log_statement_stats) |  Log the total performance statistics of the query parser, planner, and executor |
+
+[Query Minidump Log Files](#query_log) identifies configuration options and additional information related to generating query optimizer minidump log files.
+
+### <a id="logfile_manage_db"></a>Managing HAWQ Log Files 
+
+HAWQ log output tends to be voluminous, especially at higher debug levels. You should not need to save this information indefinitely. HAWQ administrators typically configure HAWQ to rotate the log files periodically so new log files are created.
+
+Daily log files are created in the `pg_log` subdirectory of the master and each segment data directory. Although log files are rolled over daily, they are not automatically truncated or deleted. Administrators must implement and run scripts to periodically clean up old log files in the `pg_log` directory of the master, standby, and every segment instance.
+
+
+## <a id="mgmtutil_log"></a>Management Utility Log Files 
+You invoke HAWQ management utilities (MU) both from the command line and when you perform cluster management activities using the Ambari user interface. Regardless of source, these utilities log information about command execution status and results. The messages are logged to both `stdout` and a log file.
```

Comment:
Not sure I like the idea of introducing the MU abbreviation.  If this isn't used elsewhere in the docs, let's just spell it out in the titles instead.